api/v1alpha1/hostedcluster_types.go

```diff
@@ -655,6 +655,14 @@ type PowerVSPlatformSpec struct {
 	//
 	// +immutable
 	ControlPlaneOperatorCreds corev1.LocalObjectReference `json:"controlPlaneOperatorCreds"`
+
+	// ClusterControlManagerCreds is a reference to a secret containing ibm cloud
+	// credentials for cluster control manager to get authenticated with ibm cloud.
+	// The secret should have exactly one key, `ibmcloud_api_key`, whose value is an apikey of user's account.
+	// This field is immutable. Once set, It can't be changed.
+	//
+	// +immutable
+	ClusterControlManagerCreds corev1.LocalObjectReference `json:"clusterControlManagerCreds"`
```

Comment:
what is a cluster control manager? did you mean cluster controller manager? how is this different than the KubeCloudControllerCreds referenced above? Can you also update the field // comments to be specific and drop the AWS mentions?

Reply:
Sry its a typo here, it should be cloud controller manager. Using KubeCloudController creds instead of this, so removed. Fixed the AWS ref in all the creds.